cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -97,9 +219,16 @@ func (o CloudControllerManagerOptions) Config() (*cloudcontrollerconfig.Config,
 	}
 
 	c := &cloudcontrollerconfig.Config{}
-	if err := o.ApplyTo(c); err != nil {
+	if err := o.ApplyTo(c, "cloud-controller-manager"); err != nil {
```

Comment:
Could you create a CloudControllerManagerUserAgent string constant in a follow-up?